src/sections/section-5.mdx

```diff
@@ -55,3 +86,8 @@ The 2019 Design Systems Survey is just the start. Sign up and receive one to two
 [2]: https://seesparkbox.com/foundry/category/design_systems "Sparbox Website Foundry Page"
 [3]: https://seesparkbox.com/work-with-us "Sparkbox Website Work With Us Page"
 [4]: http://eepurl.com/dta0D1 "Signup Form on the MailChimp Website"
+
+[5]: https://seesparkbox.com/foundry/design_systems_teams_are_part_of_the_product "Your Design System Team Is Part of the Product on the Sparkbox Foundry"
```

Comment:
Do links 4-8 get used as footnotes anywhere? I see 1-3 referenced above.

Reply:
I’m not sure about 4. I forgot about adding 5-8 earlier, those can be removed. I’ll see what is up with 4.